### ISSUE TITLE
Why the learning rate of  deformable_module is 0.1

### ISSUE THREAD

**Issue by mutianxu** [HuguesTHOMAS/KPConv-PyTorch#53]
https://github.com/HuguesTHOMAS/KPConv-PyTorch/blob/94b1d87e17ec339e2aad049ce5d5d175b9cd0db1/utils/trainer.py#L80

**Comment by HuguesTHOMAS** [comment]
It is the case in the original paper of Deformable convolution in images. From my experiment it helps a little bit.

My understanding of it is that it slows down the rate at which the deformation are learned (and thus modified) during training. Which helps the network weights "catching up" with the new dispositions that are learned, if that makes any sense to you.
